src/app/components/fees-row/components/fees-row.layout.tsx

```diff
@@ -23,7 +23,7 @@ export function FeesRowLayout(props: FeesRowLayoutProps) {
   const [_, meta] = useField('fee');
 
   return (
-    <HStack gap="space.04" width="100%" {...rest}>
+    <HStack gap="space.04" width="100%" {...rest} flexWrap={'wrap'} >
```

Comment:
I checked and this pushes the warning down so it can be seen but it doesn't look too good:

![Screenshot 2023-12-14 at 06 38 41](https://github.com/leather-wallet/extension/assets/2938440/71989c21-b734-4feb-8f98-02d657265104)

I will ask for some help from our designers with this screen

Reply:
The design team said we can should this warning fit the full width available. If you can do that and update your PR as requested I can approve and merge it. Thank you for helping!